io/disk/lvsetup.py

```diff
@@ -81,7 +81,7 @@ def setUp(self):
         self.lv_name = lv_name
         if lv_utils.lv_check(vg_name, lv_snapshot_name):
             self.cancel('Snapshot %s already exists' % lv_snapshot_name)
-        self.mount_loc = self.srcdir
+        self.mount_loc = self.workdir
```

Comment:
https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/io/disk/lvsetup.py#L72 uses `workdir` already. With this change, the test would try to mount on the `workdir` which is used as `ramdisk_basedir`.

Reply:
@clebergnu can this small change be taken care of with latest changes being pulled ?